internal/app/interactive/model/state.go

```diff
@@ -143,3 +144,14 @@ func InitProjectDir(mdl *State) {
 func (mdl *State) GetProjectPath() string {
 	return filepath.Join(mdl.ProjectDir, mdl.ProjectName)
 }
+
+func FindVacantProjectName(mdl *State) (string, bool) {
```

Comment:
Please consider raising an error here and returning string. `bool` return type is usually for predicates and it is hard to understand pair return type here